src/tests/ftest/cart/ghost_rank_rpc/one_node.py

```diff
@@ -7,18 +7,19 @@
 from cart_utils import CartTest
 
 
-class CartCoRpcOneNodeTest(CartTest):
+class CartGhostRankRpcOneNodeTest(CartTest):
     # pylint: disable=too-few-public-methods
     """Run CaRT ghost rank RPC test.
 
     :avocado: recursive
     """
 
+    @skipForTicket("CART-986")
```

Comment:
(pylint-undefined-variable) Undefined variable 'skipForTicket'

Reply:
You're missing an import:
```
from apricot import skipForTicket
```